src/AutoMapper.Collection/Configuration/GeneratePropertyMapsExpressionFeature.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoMapper.Collection.Runtime;
+using AutoMapper.EquivalencyExpression;
+using AutoMapper.Features;
+
+namespace AutoMapper.Collection.Configuration
+{
+    public class GeneratePropertyMapsnFeature : IGlobalFeature
```

Comment:
GeneratePropertyMapsExpressionFeature suppose to be name?

Reply:
Yes, that is correct, fixed.